dask_expr/_collection.py

```diff
@@ -1182,6 +1189,99 @@ def nunique(self, axis=0, dropna=True):
                 ]
             )
 
+    def info(self, buf=None, verbose=False, memory_usage=False):
+        """
+        Concise summary of a Dask DataFrame
+        """
+        from dask.utils import M
+
+        mem_use = memory_usage
+
+        if buf is None:
+            import sys
+
+            buf = sys.stdout
+        lines = [str(type(self))]
+
+        if len(self.columns) == 0:
+            lines.append(f"{type(self.index._meta).__name__}: 0 entries")
+            lines.append(f"Empty {type(self).__name__}")
+            if PANDAS_GE_150:
+                # pandas dataframe started adding a newline when info is called.
+                lines.append("")
+            put_lines(buf, lines)
+            return
+
+        # Group and execute the required computations
+        computations = {}
+        if verbose:
+            memory_usage = True
+            computations.update({"index": self.index, "count": self.count()})
+        if mem_use:
+            computations.update(
+                {"memory_usage": self.map_partitions(M.memory_usage, index=True)}
+            )
+        import dask.array as da
+
+        computations = dict(
+            zip(computations.keys(), da.compute(*computations.values()))
```

Comment:
Why do we need dask.array here?

Reply:
ya, naively just getting compute.. maybe this is better https://github.com/dask-contrib/dask-expr/pull/513/commits/a5f442bf103b59a63768811bf596680b35bb3976

Reply:
Yep, thx